src/commands/delegate/voters.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { flags as flagParser } from '@oclif/command';
+import BaseCommand from '../../base';
+import getAPIClient from '../../utils/api';
+import query from '../../utils/query';
+import { SORT_FIELDS } from '../../utils/constants';
+
+const processFlagInputs = (limitStr, offsetStr, sortStr) => {
+	const limit = parseInt(limitStr, 10);
+	const offset = parseInt(offsetStr, 10);
+	const sort = sortStr ? sortStr.trim() : undefined;
+	if (limitStr !== limit.toString() || !Number.isInteger(limit) || limit <= 0) {
+		throw new Error('Limit must be an integer and greater than 0');
+	}
+	if (
+		offsetStr !== offset.toString() ||
+		!Number.isInteger(offset) ||
+		offset < 0
+	) {
+		throw new Error('Offset must be an integer and greater than or equal to 0');
+	}
+	if (SORT_FIELDS.indexOf(sort) === -1) {
+		throw new Error(`Sort must be one of: ${SORT_FIELDS.join(', ')}`);
+	}
+
+	return {
+		limit,
+		offset,
```

Comment:
I think we should have limit and offset since these commands return potentially large amount of data in votes and voters arrays. The specified usernames in the command returns separate data sets for each delegate.  Sort is applied to voters/votes arrays for each specified delegate, so it can be useful to allow user to find information they need.